beetsplug/edit.py

```diff
@@ -302,9 +301,14 @@ def edit_objects(self, objs, fields):
                 elif choice == u'c':  # Cancel.
                     return False
                 elif choice == u'e':  # Keep editing.
-                    # Reset the temporary changes to the objects.
+                    # Reset the temporary changes to the objects. I we have a
+                    # deepcopy from above, use that, else reload from the
```

Comment:
Perhaps just call it a "copy" here, since we're not using `deepcopy` anymore.

Reply:
Right, that's a leftover from before, thanks for spotting it!